### PR TITLE
update @azure/core-rest-pipeline&@azure/core-auth&@azure-rest/core-client version

### PR DESCRIPTION
**Pull request by kazrael2119**

### CODE REVIEW

**Review by qiaozha — CHANGES_REQUESTED** [review]
as core-rest-pipeline latest beta version has some issues.

File: packages/autorest.typescript/test/version-tolerance/generated/rlc-updated/package.json [hunk 1]
```diff
@@ -57,9 +57,9 @@
   "sideEffects": false,
   "autoPublish": false,
   "dependencies": {
-    "@azure/core-auth": "^1.3.0",
-    "@azure-rest/core-client": "1.0.0-beta.9",
```
Comment (MaryGao) [inline packages/autorest.typescript/test/version-tolerance/generated/rlc-updated/package.json:61]: why we have beta? could you check?

File: packages/typespec-ts/test/modularIntegration/customClientRequestId.spec.ts [hunk 2]
```diff
@@ -65,5 +65,5 @@ describe("RequestIdClient Classical Client", () => {
     } catch (err) {
       assert.isNotNull(err);
     }
-  });
+  }).timeout(36000);
```
Comment (MaryGao) [inline packages/typespec-ts/test/modularIntegration/customClientRequestId.spec.ts:68]: change timeout in package.json not signle ut

**Review by MaryGao — APPROVED** [review]
LGTM

**Review by lirenhe — APPROVED** [review]

**Review by qiaozha — APPROVED** [review]

**Pull request merged**